Source/Core/DiscIO/Enums.cpp

```diff
@@ -555,48 +561,67 @@ const std::string& GetCompanyFromID(const std::string& company_id)
       {"JR", "Engine Software"},
       {"JS", "Digital Leisure"},
       {"JT", "Empty Clip Studios"},
+      {"JU", "Riverman Media"},
+      {"JV", "JV Games"},
       {"JW", "BigBen Interactive"},
       {"JX", "Shin'en Multimedia"},
+      {"JY", "Steel Penny Games"},
       {"JZ", "505 Games"},
+      {"K2", "Coca-Cola (Japan) Company"},
       {"K3", "Yudo"},
       {"K6", "Nihon System"},
       {"KB", "Nippon Ichi Software America"},
       {"KG", "Kando Games"},
+      {"KH", "Joju Games"},
       {"KJ", "Studio Zan"},
+      {"KK", "DK Games"},
       {"KL", "Abylight"},
       {"KM", "Deep Silver"},
+      {"KN", "Gameshastra"},
       {"KP", "Purple Hills"},
       {"KQ", "Over the Top Games"},
       {"KR", "KREA Medie"},
+      {"KT", "The Code Monkeys"},
       {"KW", "Semnat Studios"},
+      {"KY", "Medaverse Studios"},
       {"L3", "G-Mode"},
+      {"L8", "FujiSoft"},
       {"LG", "Black Bean Games"},
       {"LJ", "Legendo Entertainment"},
       {"LL", "HB Studios"},
       {"LP", "Left Field Productions"},
       {"LR", "Koch Media"},
       {"LT", "Legacy Interactive"},
+      {"LU", "Lexis Num\xc3\xa9rique"},  // We can't use a u8 prefix due to C++20's u8string
```

Comment:
What do you mean here?

Reply:
It would be convenient to write this string as `u8"Lexis Numérique"`, but if I did that, compilation would fail when C++20 is supported since it would give us a `const char8_t[]` (instead of `const char[]`), which is not compatible with `std::string`.

Reply:
TIL! That's pretty nasty... Can `char` be converted to `char8_t` though? Sounds like it may be more future proof to migrate to u8string here (once applicable) and either let it convert implicitly or prefix them all with u8.

Reply:
We would have to wait until all supported compilers have `u8string` support if we want to use `u8string`, which is going to take a while. And using `u8string` in this function doesn't really solve everything, since we have to convert it back to `string` at some point in the calling code anyway (barring a major move to `u8string` in large parts of Dolphin).